requirements/prod.txt

```diff
@@ -199,22 +190,24 @@ markupsafe==2.0.1
 oauthlib==3.1.1
     # via requests-oauthlib
 packaging==21.3
-    # via bleach
-pillow==8.4.0
+    # via
+    #   bleach
+    #   redis
+pillow==9.0.0
```

Comment:
This is a big jump. Anything to worry about?

Reply:
I suspect we aren't actually using Pillow